docs/changelog.json

```diff
@@ -1,7 +1,282 @@
 {
   "url": "https://github.com/Shopify/hydrogen/pulls?q=is%3Apr+is%3Aclosed+%5Bci%5D+release+in%3Atitle+is%3Amerged",
-  "version": "1.4.2",
+  "version": "2024.4.0",
   "releases": [
+    {
+      "title": "SFAPI 2024.4, Vite, Analytics (unstable), Remix 2.8.0, CLI improvements",
+      "version": "2024.4.0",
+      "hash": "4c80d7cd61e70c6efea931c3e14d28131850f439",
+      "pr": "https://github.com/Shopify/hydrogen/pull/1855",
+      "commit": "https://github.com/Shopify/hydrogen/pull/1855/commits/4fa33fdaab06ff5258b7c7cec4f5f24492bd7349",
+      "dependencies": {
+        "@remix-run/react": "^2.8.0",
+        "@shopify/cli-hydrogen": "$3.58.0",
+        "@shopify/hydrogen": "^2024.4.0",
+        "@shopify/remix-oxygen": "^2.0.3"
+      },
+      "devDependencies": {
```

Comment:
We also need `@shopify/hydrogen-codegen@^0.3.0` in dev dependencies. This was part of the CLI before.